engine/class_modules/sc_hunter.cpp

```diff
@@ -4838,6 +4867,17 @@ struct kill_command_t: public hunter_spell_t
         flankers_advantage.proc -> occur();
         cooldown -> reset( true );
         p() -> buffs.strength_of_the_pack -> trigger();
+
+        if ( p() -> options.t28_2pc )
+        {
+            if ( rng().roll( 0.4 ) )
+            {
+              p() -> cooldowns.wildfire_bomb -> reset( true );
+
+              // TODO: Does mad bombardier (4) trigger from only these resets or also when it comes off cd like normal?
+              p() -> buffs.mad_bombardier -> trigger();
```

Comment:
Re the 4pc question, at least the wording suggests that the buff triggers only from resets.
It also kinda makes sense mechanically - KC reset has a 40% chance to trigger a buff (https://ptr.wowhead.com/spell=363805/mad-bombardier) that actually prevents charge consumption and increases damage. So triggering it only here makes sense.